react/features/screenshot-capture/worker.ts

```diff
@@ -1,29 +1,135 @@
+import pixelmatch from 'pixelmatch';
+
 import {
-    CLEAR_INTERVAL,
-    INTERVAL_TIMEOUT,
-    SET_INTERVAL
+    CLEAR_TIMEOUT,
+    MAX_FILE_SIZE,
+    PERCENTAGE_LOWER_BOUND,
+    SEND_CANVAS_DIMENSIONS,
+    SET_TIMEOUT,
+    TIMEOUT_TICK
 } from './constants';
 
-const code = `
-    var timer;
 
-    onmessage = function(request) {
-        switch (request.data.id) {
-        case ${SET_INTERVAL}: {
-            timer = setInterval(() => {
-                postMessage({ id: ${INTERVAL_TIMEOUT} });
-            }, request.data.timeMs);
-            break;
-        }
-        case ${CLEAR_INTERVAL}: {
-            if (timer) {
-                clearInterval(timer);
+let timer: ReturnType<typeof setTimeout>;
+let height: number;
```

Comment:
I checked and ImageBitmap does expose its width and height. I think we should use those, which should simplify some things greatly.